var/spack/repos/builtin/packages/unifyfs/package.py

```diff
@@ -16,55 +16,48 @@ class Unifyfs(AutotoolsPackage):
 
     homepage = "https://github.com/LLNL/UnifyFS"
     git      = "https://github.com/LLNL/UnifyFS.git"
-    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.9.0/unifyfs-0.9.0.tar.gz"
+    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.9.1/unifyfs-0.9.1.tar.gz"
     maintainers = ['CamStan']
 
     version('develop', branch='dev')
     version('0.9.1', sha256='2498a859cfa4961356fdf5c4c17e3afc3de7e034ad013b8c7145a622ef6199a0')
-    version('0.9.0', sha256='e6c73e22ef1c23f3141646aa17058b69c1c4e526886771f8fe982da924265b0f')
```

Comment:
We discourage removing versions, especially if there is a chance someone may come along and try to install it anyway.

As an alternative you could add: `, deprecated=True` to the version statement.

Reply:
Ooh, good to know. Added this back as deprecated.